go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/dominant-strategies/go-quai
 
-go 1.19
+go 1.20
```

Comment:
Why is this needed?

Reply:
Go 1.20 allows casting a slice to an array, like so: common.AddressBytes([]byte{1})

Reply:
mmm yes, I see